RepositoryCompiler/CodeModel/CodeParsers/CSharp/CSharpMetricCalculator.cs

```diff
@@ -40,41 +40,43 @@ public CaDETClassMetrics CalculateClassMetrics(CaDETClass parsedClass)
         private double? GetTightClassCohesion(CaDETClass parsedClass)
         {
             int N = GetNumberOfMethodsDeclared(parsedClass);
+           
             double NP = (N * (N - 1)) / 2;
             if (NP == 0) return null;
 
-            return NumberOfDirectConnections(parsedClass) / NP;
+            return GetNumberOfDirectConnections(parsedClass) / NP;
         }
 
-        private int NumberOfDirectConnections(CaDETClass parsedClass)
+        private int GetNumberOfDirectConnections(CaDETClass parsedClass)
         {
-            int mapIndex = 1;
-            var map = new Dictionary<int, List<CaDETMember>>();
+            var ownAccessedFieldsAndAccessors = new Dictionary<string, List<CaDETMember>>();
```

Comment:
This is a much better name that completely conveys meaning - I cannot overstate how easier it is to understand this code just with this one rename. Keep it up :)